src/components/media-loader.js

```diff
@@ -65,9 +73,24 @@ AFRAME.registerComponent("media-loader", {
           },
           { once: true }
         );
-        this.el.setAttribute("image-plus", { src: raw, contentType });
+        let blobUrl;
+        if (token) {
+          const imageResponse = await fetch(raw, {
+            method: "GET",
+            headers: { Authorization: `Token ${token}` }
+          });
+          const blob = await imageResponse.blob();
+          blobUrl = window.URL.createObjectURL(blob);
```

Comment:
Don't think these blob urls ever get cleaned up. When to do that is a bit tricky....